packages/coordinator/tests/services/verification/signature-verification.spec.ts

```diff
@@ -96,12 +96,42 @@ describe("verifyPrescriptionSignature", () => {
 
   describe("Signature is invalid", () => {
     const validSignature = TestResources.parentPrescriptions.validSignature.ParentPrescription
+    const valid256Signature = TestResources.parentPrescriptions.sha256Signature.ParentPrescription
 
     test("passes if prescription has valid Signature that matches prescription", async () => {
       const result = await verifyPrescriptionSignature(validSignature, logger)
       expect(result).not.toContain("Signature is invalid")
     })
 
+    test("passes if prescription signature method algorithm that references SHA-256 matches prescription", async () => {
+      const result = await verifyPrescriptionSignature(valid256Signature, logger)
+      expect(result).not.toContain("Signature is invalid")
+    })
+
+    test("passes if prescription signature method algorithm that references not SHA-256 or SHA-1 matches prescription",
+      async () => {
+        const clonePrescription = clone(validSignature)
+        const signatureRoot = extractSignatureRootFromParentPrescription(clonePrescription)
+        signatureRoot.Signature.SignedInfo.SignatureMethod._attributes.Algorithm =
+        "http://www.w3.org/2000/09/xmldsig#"
+        signatureRoot.Signature.SignedInfo.Reference.DigestMethod._attributes.Algorithm =
+        "http://www.w3.org/2000/09/xmldsig#"
+        const result = await verifyPrescriptionSignature(clonePrescription, logger)
+        expect(result).not.toContain("Signature is invalid")
+      })
+
+    test("fails if prescription signature method algorithm that references SHA-224 does not match prescription",
```

Comment:
As above. A clearer name, I think.
Maybe "fails if prescription signature is valid but method algorithm references incorrect encoding"
Again, could just be my lack of knowledge on this ticket.

Reply:
The proposed change has been applied.